README.md

```diff
@@ -0,0 +1,158 @@
+# FakeFish
+
+> **WARNING**: The work exposed here is not supported in any way by Red Hat, this is the result of exploratory work. Use at your own risk.
+
+`FakeFish` is a flask based app that exposes a RedFish-like API with a set of limited endpoints that allow the deployment of OpenShift nodes via the Metal3 operator on hardware that doesn't support RedFish or doesn't follow the RedFish standard.
+
+The way it works is by running a set of scripts that interact with the hardware using vendor tools/other methods while exposing a fake RedFish API that Metal3 can query.
+
+The [app/](./app/) directory contains the FakeFish application. Inside the `app` directory we can find the [custom_scripts](./app/custom_scripts/) folder where we will find 5 scripts:
+
+|Script|What should it do?|
+|------|----------------|
+|`poweron.sh`|Power on the server|
+|`poweroff.sh`|Power off the server|
+|`bootfromcdonce.sh`|Set server to boot from virtual CD once|
+|`mountcd.sh`|Mount the iso received in the server's virtual CD|
+|`unmountcd.sh`|Unmount the iso from the server's virtual CD|
+
+The script names can't be changed, and the scripts in the `custom_scripts` folder are just placeholders. Users need to implement their own scripts, we will not maintain/add providers to this project.
+
+Users can take a look at the [dell_scripts](./dell_scripts/) folder where they can find the scripts we used for our testing with DELL hardware. Remember that these scripts are unsupported/unmaintained and should be taken as a reference, nothing else.
+
+A [Containerfile](./app/Containerfile) is included, so users can build their own container image out of it. We published an image using the bits of the first release at `quay.io/mavazque/fakefish:v0`.
```

Comment:
We'll need to change this. I don't know if we should keep publishing images at all (rather than allowing the users to publish theirs). @rhjanders?

Reply:
@dtantsur here's my current thinking:
* initially, let's not worry about the images and assume users will make their own (this is to get to a good state asap)
* in general, perhaps it's beneficial to have images available to make getting started it as easy as possible
* however for that to work we need to think through how the images are built and kept up to date (better have no images available than to have old images) and that's adding complexity which is likely not needed at this stage.
WDYT?